db/migrate/20170626222246_create_access_logs.rb

```diff
@@ -0,0 +1,8 @@
+class CreateAccessLogs < ActiveRecord::Migration[5.1]
+  def change
+    create_table :access_logs do |t|
```

Comment:
The `access_logs` table is already being created in the above migration. This one should probably be deleted.